sources/packages/backend/apps/api/src/services/application/application.service.ts

```diff
@@ -553,6 +558,9 @@ export class ApplicationService extends RecordDataModelService<Application> {
         location: {
           id: true,
           name: true,
+          institution: {
```

Comment:
Are we adding this extra select and the `location` on `relations` because of the where condition or is it needed to be returned in the method now?

Reply:
returned as part of where condition

Reply:
@guru-aot if that is the case, I believe we can remove the `institution:true` in the relations and leave it as it is. Because the where will work without relevance in relations.